package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "api-docs",
-  "version": "1.0.0",
-  "description": "",
+  "private": true,
```

Comment:
Because this is not a package, always add `private`, someone can pack and push to their account.